operators/pkg/tenant-controller/tenant_controller.go

```diff
@@ -517,6 +532,32 @@ func (r *TenantReconciler) createOrUpdateTnPersonalNFSVolume(ctx context.Context
 			return err
 		}
 		klog.Infof("PVC Secret for tenant %s %s", tn.Name, pvcSecOpRes)
+
+		val, ok := pvc.Labels[ProvisionJobLabel]
+		if !ok || val != ProvisionJobValueOk {
```

Comment:
split the logic based on check. if no labels create, if label is not ok, check if it's done, etc